css/module.css

```diff
@@ -1,66 +1,90 @@
-input[type="text"],
-input[type="email"],
-input[type="tel"],
-textarea {
-  height: 1.5em;
-  width: 30em;
-  background-color: #fff;
-  border: 1px solid #D1D1D1;
-  border-radius: 5px;
-  margin-bottom: 1em;
-  color: black;
-}
+/* ------- Contact form module ------- */
```

Comment:
I see that you've improved the construction of your modules in this iteration. I will regrade the previous assignment that I had made a comment on for you to revise..